client/extensions/woocommerce/woocommerce-services/lib/initialize-labels-state/index.js

```diff
@@ -42,6 +42,35 @@ export default data => {
 	const hasOriginAddress = addressFilled( formData.origin );
 	const hasDestinationAddress = addressFilled( formData.destination );
 
+	const customsItemsData = {};
+	forEach( formData.selected_packages, ( { items } ) => {
+		items.map(
+			( {
+				product_id,
+				weight,
+				name,
+				attributes,
+				description,
+				value,
+				hs_tariff_number,
+				origin_country,
+				parent_product_id,
+			} ) => {
+				const attributesStr = attributes ? ' (' + attributes + ')' : '';
+				const defaultDescription = name.substring( name.indexOf( '-' ) + 1 ).trim() + attributesStr;
+				customsItemsData[ product_id ] = {
```

Comment:
We should have some unit tests here, for documentation and to make sure we are indeed returning the values we expect now and in later changes.